DOCPLAN-164: CQA ocp virt storage docs (#110155)
openshift/openshift-docs: virt/storage/install-configure-fusion-access-san.adoc

toc::[]

[role="_abstract"]

> You can use {IBMFusionFirst} with {VirtProductName} to configure SAN-based storage for virtual machines. This involves installing the required operator and setting up the storage cluster and file systems.

Reviewer: ```suggestion
You configure SAN-based storage for virtual machines by using {IBMFusionFirst} with {VirtProductName}. You must install the required Operator and set up the storage cluster and file systems.
```

- Suggested editing, not mandatory.
- [Operator needs to be capitalized](https://redhat-documentation.github.io/supplementary-style-guide/#operator) but it might be better to put the actual Operator name here so that it's clearer?

include::modules/virt-troubleshoot-fusion-access-san.adoc[leveloffset=+1]

include::modules/virt-fusion-access-san-release-updates.adoc[leveloffset=+1]

[id="additional-resources_install-configure-fusion-access-san"]

> == Additional resources

Reviewer: ```suggestion
[role="_additional-resources"]
[id="additional-resources_{context}"]
== Additional resources
```
{context} will pick up the right context from the assembly, you can just use the anchoring style in https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies
